mappings/net/minecraft/world/chunk/WrapperProtoChunk.mapping

```diff
@@ -1,8 +1,12 @@
-CLASS net/minecraft/class_2821 net/minecraft/world/chunk/ReadOnlyChunk
-	COMMENT Represents a read only view of a world chunk used in world generation.
+CLASS net/minecraft/class_2821 net/minecraft/world/chunk/WrapperProtoChunk
+	COMMENT Represents a view of a world chunk used in world generation.
+	COMMENT
+	COMMENT Can be read only.
```

Comment:
hmm, i think when this is supposedly read-only, it still supports a few functions. this is like a chunk that only support (thread safe?) certain operations to avoid unintended pollution.